website/docs/r/schematics_agent.html.markdown

```diff
@@ -0,0 +1,254 @@
+
+* `id` - The unique identifier of the schematics_agent.
+* `agent_crn` - (String) The agent crn, obtained from the Schematics agent deployment configuration.
+* `created_at` - (String) The agent creation date-time.
+* `creation_by` - (String) The email address of an user who created the agent.
+* `recent_deploy_job` - (List) Post-installations checks for Agent health.
+Nested scheme for **recent_deploy_job**:
+	* `agent_id` - (String) Id of the agent.
+	* `agent_version` - (String) Agent version.
+	* `is_redeployed` - (Boolean) True, when the same version of the agent was redeployed.
+	* `job_id` - (String) Job Id.
+	* `log_url` - (String) URL to the full agent deployment job logs.
+	* `status_code` - (String) Final result of the agent deployment job.
+	  * Constraints: Allowable values are: `pending`, `in-progress`, `success`, `failed`.
+	* `status_message` - (String) The outcome of the agent deployment job, in a formatted log string.
+	* `updated_at` - (String) The agent deploy job updation time.
+	* `updated_by` - (String) Email address of user who ran the agent deploy job.
+* `recent_health_job` - (List) Agent health check.
+Nested scheme for **recent_health_job**:
+	* `agent_id` - (String) Id of the agent.
+	* `agent_version` - (String) Agent version.
+	* `job_id` - (String) Job Id.
+	* `log_url` - (String) URL to the full health-check job logs.
+	* `status_code` - (String) Final result of the health-check job.
+	  * Constraints: Allowable values are: `pending`, `in-progress`, `success`, `failed`.
+	* `status_message` - (String) The outcome of the health-check job, in a formatted log string.
+	* `updated_at` - (String) The agent health check job updation time.
+	* `updated_by` - (String) Email address of user who ran the agent health check job.
+* `recent_prs_job` - (List) Run a pre-requisite scanner for deploying agent.
+Nested scheme for **recent_prs_job**:
+	* `agent_id` - (String) Id of the agent.
+	* `agent_version` - (String) Agent version.
+	* `job_id` - (String) Job Id.
+	* `log_url` - (String) URL to the full pre-requisite scanner job logs.
+	* `status_code` - (String) Final result of the pre-requisite scanner job.
+	  * Constraints: Allowable values are: `pending`, `in-progress`, `success`, `failed`.
+	* `status_message` - (String) The outcome of the pre-requisite scanner job, in a formatted log string.
+	* `updated_at` - (String) The agent prs job updation time.
+	* `updated_by` - (String) Email address of user who ran the agent prs job.
+* `system_state` - (List) Computed state of the agent.
+Nested scheme for **system_state**:
+	* `status_code` - (String) Agent Status.
+	  * Constraints: Allowable values are: `error`, `normal`, `in_progress`, `pending`, `draft`.
+	* `status_message` - (String) The agent status message.
+* `updated_at` - (String) The agent registration updation time.
+* `updated_by` - (String) Email address of user who updated the agent registration.
+
```

Comment:
Below mentioned sections are generic and is available in provider docs..
```
## Provider Configuration
### Static credentials
### Environment variables
```
kindly remove these sections from the docs

Reply:
done